Firmware/cmdqueue.cpp

```diff
@@ -442,6 +442,8 @@ void get_command()
         bool allow_when_stopped = false;
         if(strncmp_P(cmd_start, PSTR("M310"), 4) == 0)
             allow_when_stopped = true;
+        if(strcmp_P(cmd_start, PSTR("M500")) == 0)
```

Comment:
may be this rules out g-code lines which have some whitespace characters after `M500`:
e.g.:
```
M500....
```
Do we want to support that?

Reply:
I really like to avoid M500 here. This is not a safe command to allow in an emergency, and any command added here can/will break the serial protocol if seen.

IMHO the stopped state should be cleared from the printer, after which you can send an M500 as usual.

Reply:
I don't think we have a proper way to do this if there is no active print.

You can clear it normally via stop or start print. Maybe we need a "Clear error" for the "no print" case, such as usb.